manage-mypy.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from subprocess import run
+from typing import List
+
+import click
+
+file_path = Path(__file__)
+here = file_path.parent
+exclusion_file = here / "mypy-exclusions.txt"
+
+
+def build_exclusion_list() -> List[str]:
+    # Create content for `mypy-exclusions.txt` based on a `mypy` run with `mypy.ini.template`
+    command = ["python", "activated.py", "mypy", "--config-file", "mypy.ini.template"]
+    lines = run(command, capture_output=True).stdout.decode("utf8").splitlines()
+    return sorted({".".join(Path(line[: line.find(".py")]).parts) for line in lines[0 : len(lines) - 1]})
+
+
+@click.group()
+def main() -> None:
+    pass
+
+
+@main.command()
+@click.option("--check-exclusions/--no-check-exclusions", show_default=True, envvar="CHIA_MANAGE_MYPY_CHECK_EXCLUSIONS")
+def build_mypy_ini(check_exclusions: bool = False) -> None:
+    if not exclusion_file.exists():
+        raise click.ClickException(f"{exclusion_file.name} missing, run with `{file_path.name}`")
+    if check_exclusions:
+        updated_exclusions = build_exclusion_list()
+        # Compare the new old content with the new content and fail if some file without issues is excluded.
+        old_exclusions = exclusion_file.read_text().splitlines()[1:]
+        if updated_exclusions != old_exclusions:
+            fixed = "\n".join([f"  -> {entry}" for entry in set(old_exclusions) - set(updated_exclusions)])
+            if len(fixed) > 0:
+                raise click.ClickException(
+                    f"The following fixed files need to be dropped from {exclusion_file.name}:\n{fixed}"
+                )
+
+    # Create the `mypy.ini` with all entries from `mypy-exclusions.txt`
+    exclusion_file_content = exclusion_file.read_text().splitlines()
+    exclusion_lines = [line for line in exclusion_file_content if not line.startswith("#") and len(line.split()) > 0]
+    exclusion_section = f"[mypy-{','.join(exclusion_lines)}]"
+    mypy_config_data = (here / "mypy.ini.template").read_text().replace("[mypy-chia-exclusions]", exclusion_section)
```

Comment:
Hm, why? I don't care much and i will change it but i still wonder why because the `/` is pretty nice actually and much shorter :)

Reply:
Exactly because of this awkward inconsistency where you thrash between parenthesizing it and not depending on whether you do something with it or not.  I forget what else.  I mean dividing paths is just a weird concept.  `:]`  Calling a method is pretty natural.  But yeah, of all goofy overloads, `/` for paths is _relatively_ nifty.